libs/designsystem/calendar/src/calendar.component.html

```diff
@@ -24,7 +24,7 @@
       [noDecoration]="true"
       class="no-margin"
     >
-      <kirby-icon name="arrow-more"></kirby-icon>
+      <kirby-icon name="arrow-more" aria-label="Next month"></kirby-icon>
```

Comment:
Opted to add the aria-label (that also needs translation!) on the icon, not the button, as I think that makes most sense. It is the icon we want to describe, as it is added in place of text content.